db/seeds.rb

```diff
@@ -5,3 +5,45 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+dogs = [
+  {
+    id: 1,
```

Comment:
Your seeds should not have an id. The id will be created with the data is added to the db with the Active Record create method on line 47.